Resources/Prototypes/Entities/Mobs/Cyborgs/borg_chassis.yml

```
@@ -135,6 +135,10 @@
     templateId: borgShort
   - type: SiliconLawProvider # Delta-V - Adds custom lawset for Engineering Cyborg
     laws: Engineer
+  - type: ShowHealthBars
+    damageContainers:
+    - Silicon
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 6 but found 4

(indentation)